plugins/recruit-resources/src/index.ts

```diff
@@ -114,26 +114,66 @@ export async function queryApplication (client: Client, search: string): Promise
   }))
 }
 
-export async function getActiveTalants (filter: Filter, onUpdate: () => void): Promise<Array<Ref<Doc>>> {
+async function getActiveTalants (filter: Filter, onUpdate: () => void): Promise<Array<Ref<Doc>>> {
   const promise = new Promise<Array<Ref<Doc>>>((resolve, reject) => {
     let refresh: boolean = false
-
-    const lq = FilterQuery.getLiveQuery(filter.index)
+    const indexKey = `${filter.index}:${filter.mode}`
```

Comment:
Why we need it? It can cause memory leak

Reply:
I do not see it removed at all for any, will check and update.

Reply:
Reverted.